spec/API_specification/data_type_functions.md

```diff
@@ -0,0 +1,81 @@
+# Data type functions
+
+> Array API specification for data type functions.
+
+A conforming implementation of the array API standard must provide and support the following data type functions.
+
+<!-- NOTE: please keep the constants in alphabetical order -->
+
+## Objects in API
+
+(finfo)=
+### finfo(type)
```

Comment:
`type` should be a positional argument here, so this should be `finfo(type, /)`. 

And same for `iinfo`.